src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -98,6 +98,18 @@ def delete_fs_volume(self, volname, confirm):
                 "that is what you want, re-issue the command followed by " \
                 "--yes-i-really-mean-it.".format(volname)
 
+        ret, out, err = self.mgr.check_mon_command({
+            'prefix': 'config get',
+            'key': 'mon_allow_pool_delete',
+            'who': 'mon',
+            'format': 'json',
+        })
+        mon_allow_pool_delete = json.loads(out)
+        if not mon_allow_pool_delete:
+            return -errno.EPERM, "", "pool deletion is disabled; you must first " \
+                "set the mon_allow_pool_delete config option to true before volumes " \
+                "can be deleted"
```

Comment:
This check breaks a lot of the volumes tests. We need to set `mon_allow_pool_delete` to `True` for all tests. Suggest putting

`self.config_set('mon', 'mon_allow_pool_delete', True)`

in `setUp`.

See: /ceph/teuthology-archive/pdonnell-2020-07-01_06:37:23-fs-wip-pdonnell-testing-20200701.033411-distro-basic-smithi/5193054/teuthology.log